apps/pie-docs/src/assets/styles/components/_contentPage.scss

```diff
@@ -39,29 +39,32 @@
 }
 
 .c-contentPage-header--hero {
-    background-image: var(--hero-img-src);
-    background-size: cover;
-    background-repeat: no-repeat;
-    background-position-x: 5%;
+    background-color: f.$color-support-brand-02;
 
-    @include f.media('>=narrowMid') {
-        background-position: center;
+    .c-contentPage-header-description {
+        margin-top: 0;
+        margin-bottom: f.$spacing-f;
     }
 
-    @include f.media('>=#{v.$wide-screen-content-max-width}') {
-        background-size: v.$wide-screen-content-max-width;
-        background-color: f.$color-container-strong;
-    }
-}
+    @include f.media('>=mid') {
+        display: flex;
+        gap: f.$spacing-f;
 
-.c-contentPage-header--hero .c-contentPage-header-heading,
-.c-contentPage-header--hero .c-contentPage-header-description {
-    @include f.media('>=narrowMid') {
-        max-width: 60%;
+        &:after {
+            content: var(--hero-img-src);
+        }
     }
 
-    @include f.media('>wide') {
-        max-width: 55%;
+    .c-contentPage-header-cta {
+        display: inline-block;
+        background-color: f.$color-interactive-brand;
+        color: f.$color-container-default;
+        font-size: f.$font-interactive-m-size;
+        font-weight: f.$font-interactive-m-weight;
+        border-radius: f.$radius-rounded-e;
+        padding: 10px f.$spacing-e;
```

Comment:
it's uncommon for us to deviate away from the regular padding values we have (multiples of 8) - is this definitely what we want? :)

Reply:
We already have a button/cta style from Fozzie - if this is that then I'd recommend using `o-btn` here

Reply:
Oh, i didn't know about this, changed!